src/pages/studyView/table/MultiSelectionTable.tsx

```diff
@@ -210,9 +220,37 @@ export class MultiSelectionTable extends React.Component<
                 render: (data: MultiSelectionTableRow) => {
                     return (
                         <div className={styles.labelContent}>
-                            <EllipsisTextTooltip
-                                text={data.label}
-                            ></EllipsisTextTooltip>
+                            <EllipsisTextTooltip text={data.label} />
```

Comment:
see if we can figure out a way to keep this class generic (not have knowledge of genepanelness